app/views/article.html

```diff
@@ -73,7 +73,7 @@
       "isPartOf": {
         "@type": ["CreativeWork", "Product"],
         "name" : "Scenic News",
-        "productID": "<% publicationId %>:news"
+        "productID": "<% config.publicationId %>:news"
```

Comment:
Let's also add `config.name` in the title.

Reply:
done